cypress/integration/creatorOnboardingFlows/creatorSettings.spec.js

```diff
@@ -120,4 +89,86 @@ describe('Creator Settings Page', () => {
     cy.findByRole('button', { name: 'Finish' }).click();
     cy.url().should('equal', `${baseUrl}admin/creator_settings/new`);
   });
+
+  context('color contrast ratios', () => {
+    it('should show an error when the constrast ratio of a brand color is too low', () => {
+      const lowContrastColor = '#a6e8a6';
+
+      cy.findByLabelText(/^Brand color/)
+        .clear()
+        .type(lowContrastColor)
+        .trigger('change');
```

Comment:
The reason the `.trigger('change')` is required is that the colour only changes once focus is lost from the input.

```suggestion
        .blur();
```